django_replicated/dbchecker.py

```diff
@@ -5,15 +5,16 @@
 import socket
 from functools import partial
 
+import django
 from django.conf import settings
 from django.core.cache import DEFAULT_CACHE_ALIAS
 from django.db import connections
 
-try:
+if django.__version__ < '1.7':
```

Comment:
I think it's better to use `django.get_version()` call instead of `__version__`.

Reply:
At first, I was not sure about comparing version tuples (which one can get from `get_version` AFAIK), but i guess there is not much difference with string comparation, so i'll switch to function

Reply:
Ok, get_version returns string too, i mixed up with django.VERSION, which returns tuples.

Reply:
Fixed